atd-vze/src/views/Crashes/VictimNameField.js

```diff
@@ -39,7 +39,7 @@ const VictimNameField = ({
   // Format name by concatenating first, middle, last or returning NO DATA
   const formatName = () => {
     var concatenatedName = "";
-    Object.keys(nameFieldConfig.subfields).map(field => {
+    Object.keys(nameFieldConfig.subfields).forEach(field => {
```

Comment:
just getting rid of a console warning here, not trying to return an array so `forEach` is better